src/lib/mdns/tests/TestServiceNaming.cpp

```diff
@@ -153,11 +153,85 @@ void TestMakeServiceNameSubtype(nlTestSuite * inSuite, void * inContext)
     NL_TEST_ASSERT(inSuite, strcmp(buffer, "") == 0);
 }
 
+void TestMakeCommissionableNodeServiceTypeName(nlTestSuite * inSuite, void * inContext)
+{
+    // TODO(cecille): These need to be changed to remove leading zeros
+    constexpr size_t kSize = 128;
+    char buffer[kSize];
```

Comment:
Add a negative case where the buffer is too small and a case where the buffer is exactly as needed?

Reply:
Added in next commit. Good call.